### PR TITLE
Add Gauntlet X and Sirius sets

### LINKED ISSUE #17: Gauntlet runmons
hhhhhhh

### PR DESCRIPTION
**Pull request by FrostedGeulleisia**
Moth had to be altered since Quiver Dance doesn't exist, if you have other suggestions - feel free to let me know.
Effectively closes #17.

### CODE REVIEW

File: Runmon/47_Sirius.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+displayname: "'HH   "
+setname: TPP Sirius
+tags: [runmon, in-game]
+item: [Leftovers]
+ability: [Effect Spore]
+moves:
+    - [Focus Punch]
+    - [Leech Seed]
+    - [Substitute]
+    - [Spore (=18)]
```
Comment (mathfreak231) [inline Runmon/47_Sirius.yaml:12]:
```suggestion
    - [Spore (+1)]
```

Comment (FrostedGeulleisia) [reply Runmon/47_Sirius.yaml:12]: Well, you said yourself in the linked issue that Hippopotas should be included too, and Wobbuffet was HoFed, even if it didn't battle much.

**Comment by mathfreak231** [comment]
I said "Hippopotas...may or may not be notable"

I should have made it clearer that I wanted a case for its notability if you were going to add it in

**Comment by mathfreak231** [comment]
Apparently we had it for "a few gyms" early game. Works for me I guess, meh maybe someday we'll have actual written guidelines for runmons

Just make the change on Spore from absolute PP to PP ups and you're good

**Comment by FrostedGeulleisia** [comment]
Hey at least i'm learning Kappa

**Comment by mathfreak231** [comment]
thank

**Pull request merged**